### PR TITLE
Add deprecation message to Calculate Paalman Pings to get user feedback

### PR DESCRIPTION
**Pull request by robertapplin**
This is not required to be merged before beta testing starts
---------

### Description of work
This PR adds a deprecation message to the Calculate Paalman Pings tab of Inelastic Corrections. It says:
```
Deprecation warning: This feature has been deprecated as of version 6.10. If you still use this feature, please inform a Mantid developer. Assuming no user feedback, this feature will be removed in version 6.12 (in two minor releases time). Please note that the "Calculate Monte Carlo Absorption" tab offers a far better version of this feature.
```

Part of #36365

### To test:
1. Open Inelastic Corrections
2. Go to the Calculate Paalman Pings tab
3. You should see a deprecation warning:
image

### Reviewer

Please comment on the points listed below ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)).
**Your comments will be used as part of the gatekeeper process, so please comment clearly on what you have checked during your review.** If changes are made to the PR during the review process then your final comment will be the most important for gatekeepers. In this comment you should make it clear why any earlier review is still valid, or confirm that all requested changes have been addressed.

#### Code Review

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?
- Do the release notes conform to the [release notes guide](https://developer.mantidproject.org/Standards/ReleaseNotesGuide.html)?

#### Functional Tests

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### Gatekeeper

If you need to request changes to a PR then please add a comment and set the review status to "Request changes". This will stop the PR from showing up in the list for other gatekeepers.

### CODE REVIEW

**Comment by SpencerHowells** [comment]
@robertapplin Is it worth mentioning that Calculate Monte Carlo Absorption is a much better version

File: qt/scientific_interfaces/Inelastic/Corrections/CalculatePaalmanPings.ui [hunk 1]
```diff
@@ -44,7 +44,7 @@
...
-&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Deprecation warning:&lt;/span&gt; This feature has been deprecated as of version 6.10. If you still use this feature, please inform a Mantid developer. Assuming no user feedback, this feature will be removed in version 6.12 (in two minor releases time).&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
+&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Deprecation warning:&lt;/span&gt; This feature has been deprecated as of version 6.10. If you still use this feature, please inform a Mantid developer. Assuming no user feedback, this feature will be removed in version 6.12 (in two minor releases time). Please note that the &amp;quot;Calculate Monte Carlo Absorption&amp;quot; tab offers a far better version of this feature.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
```
Comment (adriazalvarez) [inline qt/scientific_interfaces/Inelastic/Corrections/CalculatePaalmanPings.ui:47]: _please inform a Mantid developer._ -> Is it worth adding here support mail?

_Assuming no user feedback, this feature will be removed in version 6.12 (in two minor releases time)*_-> Can this be : 
 _Assuming no user feedback, this feature will be removed two minor versions from now (in version 6.12)_

**Review by adriazalvarez — CHANGES_REQUESTED** [review]
Tested opening corrections interface. Warning message appears correctly formatted. 
Just added minor comment.

**Review by adriazalvarez — DISMISSED** [review]
Thanks for the changes.

**Pull request merged**